website/content/en/docs/building-operators/ansible/testing-guide.md

```diff
@@ -13,7 +13,7 @@ To begin, you should have:
 - Docker installed and running
 - [Molecule](https://github.com/ansible/molecule) >= v3.0
 - [Ansible](https://github.com/ansible/ansible) >= v2.9
-- [The OpenShift Python client](https://github.com/openshift/openshift-restclient-python) >= v0.8
+- [The Python Kubernetes client](https://github.com/kubernetes-client/python)
```

Comment:
Is there a specific version that should be used or is "latest" okay?

Reply:
I think this is okay for now and we can update it with the version that we use in your [PR](https://github.com/operator-framework/operator-sdk/pull/6483)